### ISSUE TITLE
stream support enhancements

### ISSUE THREAD

**Issue by krm1312** [joeferner/redis-commander#369]
Basic streams support exists, however, it appears to just be displaying the first field of each item.  Ideally it would show each item as a list of name/value pairs like hashes are displayed.

![image](https://user-images.githubusercontent.com/532397/66233649-da75c400-e6b9-11e9-9401-308a5445cf3f.png)

Each item above has a half-dozen or more name/value pairs.

The add new entry button seems to be making an assumption about a single field as well.

**Comment by sseide** [comment]
yes, you are right, would probably be helpful. But stram support is quite new here and nobody needed it up to now. If you like to contribute...
